.github/workflows/e2e_tests.yml

```diff
@@ -70,3 +81,11 @@ jobs:
             tests/e2e/docker/videos
             tests/e2e/docker/screenshots
         continue-on-error: true
+
+      - name: Archive e2e artifacts to GCS
+        uses: google-github-actions/upload-cloud-storage@v2
+        if: github.event_name == 'schedule'
+        with:
+          path: 'tests/e2e'
+          destination: 'github-artifacts/${{ github.repository }}/${{ github.run_id }}/${{ github.event.repository.updated_at }}'
```

Comment:
We may want to add a log entry here that prints the upload path which someone with permissions can click to open and watch the recordings.

Reply:
it already the paths of all uploaded files in the logs: 
![image](https://github.com/Agoric/dapp-econ-gov/assets/8860764/3242fe0c-0019-474b-9b0f-3b2339c3201e)

Reply:
Is there a possibility to print a HTTP path to the video folder of this job?
Example:
https://console.cloud.google.com/storage/browser/github-artifacts/Agoric/dapp-econ-gov/8817132062/2024-04-24T12:12:59Z/e2e/docker/videos

Reply:
This UX consideration, when Brendan comes to this job to check the videos, they might not know the full path and gs:// might be strange to them.

Reply:
updated
![image](https://github.com/Agoric/dapp-econ-gov/assets/8860764/c1a7e79a-4002-422d-a4df-c8b466b9921d)